app/frontend/src/features/messages/messagelist/InfiniteMessageLoader.tsx

```diff
@@ -32,18 +32,20 @@ const useStyles = makeStyles((theme) => ({
 
 interface InfiniteMessageLoaderProps {
   earliestMessageId?: number;
-  latestMessageId?: number;
+  latestMessage?: { messageId: number; authorUserId: number };
+  currentUserId?: number;
   fetchNextPage: () => void;
   isFetchingNextPage: boolean;
   hasNextPage: boolean;
   isError: boolean;
   className?: string;
-  children: ReactChild;
+  children: ReactNode;
 }
 
 export default function InfiniteMessageLoader({
   earliestMessageId,
-  latestMessageId,
+  latestMessage,
+  currentUserId,
```

Comment:
Instead of drilling props, use `const currentUserId = useAuthStore().authState.userId`

Reply:
done 👍